### DISCUSSION TITLE
Plans for pulling latest changes from original diffuser repo

### DISCUSSION THREAD

**Discussion by jysung** [TheLastBen/fast-stable-diffusion#544] [Q&A]
@TheLastBen Do you have plans to pull the latest commits from upstream?
I saw that HuggingFace has a bunch of different optimization that uses lower memory and such.
How are your changes in your `diffuser` repo different?

**Comment by TheLastBen** [comment]
I have tried with the latest diffusers repo, and it consumes more VRAM, I'm still working on finding out the cause
